### DISCUSSION TITLE
switch to Qt? why not Gtk4

### DISCUSSION THREAD

**Discussion by laoshaw** [audacity/audacity#5489] [Development]
Just learned audacity might switch to Qt, why not Gtk as wxWidgets has gtk support(on linux) that could be easier for migration comparing to a whole new Qt?

**Comment by shoogle** [comment]
A bit late for this since work is already well underway to support Qt, but the reasons are:

- Qt has much better support for macOS than GtK.
- Audacity is a C++ program, whereas GTK is plain C.
- [MuseScore](https://github.com/musescore/MuseScore) (another open source app that we develop) already uses Qt, and we plan to share many interface components between the two applications.
